cdap-ui/app/directives/complex-schema/complex-schema.less

```diff
@@ -77,6 +77,7 @@ my-complex-schema {
     .input-column {
       float: left;
       width: ~"calc(100% - @{checkbox-column-width} - @{type-column-width})";
+      min-width: 55px;
```

Comment:
this will cause the row to wrap. I think you should just remove this since you already reduce the type column width